tests/compiletest.rs

```diff
@@ -135,7 +139,12 @@ fn run_tests(mode: Mode, path: &str, target: &str, with_dependencies: bool) -> R
             "run".into(), // There is no `cargo miri build` so we just use `cargo miri run`.
         ];
     }
-    ui_test::run_tests(config)
+    ui_test::run_tests_generic(
+        config,
+        |path| path.extension().map(|ext| ext == "rs").unwrap_or(false),
+        |_, _| None,
```

Comment:
Here I wish we had named arguments...

Reply:
For now, I think comments about what the arguments here are doing are sufficient.